health_vara.py

```diff
@@ -41,10 +45,62 @@ def default_name_representation():
         # 'cjk'  => 'CJK: Family+Given'
         return 'gf'
 
+    @classmethod
+    def set_mobile(cls, parties, _field_name, value):
+        Party = Pool().get('party.party')
+        # treat whitespace values the same as None
+        value = None if (value is None) or (value == '') or (value.isspace()) else value
+
+        for party in parties:
+            mobile = next((mechanisms for
```

Comment:
I think we should iterate over the list of all contact mechanisms instead of picking up the first item in the list.

Reply:
We pick the first one with type `mobile`
I'm not sure how this looks if we anticipate they have more than one mobile number.
I guess on Patient view I would have to change from a text field to a list?

Reply:
I'm not sure we'd want to set all Mobile's to the same number, maybe I could raise an exception, or set everything to inactive and create ours as the new and only active value

Reply:
Ah, right. I hadn't though it through. We don't have multiple mobiles on the Patient view.